server/views/pages/activities/create-an-activity/pay.njk

```diff
@@ -91,12 +91,11 @@
                           '
                 }) }}
 
-
                 {% if payRateType === "single" %}
-                    {% if session.req.params.mode == 'edit' and hasAllocations %}
-                        <input type='hidden' name='incentiveLevel' value='{{ iep }}' >
-                    {% elseif session.req.params.mode == 'create' and session.createJourney.incentiveLevel !== null %}
+                    {% if session.req.params.mode == 'create' and session.createJourney.incentiveLevel !== null %}
                         <input type='hidden' name='incentiveLevel' value='{{ session.createJourney.incentiveLevel }}' >
+                    {% elseif session.req.params.mode == 'edit' and (hasAllocations or iep) %}
```

Comment:
Checks is iep is present on the query, if it is we shouldn't be asking for it again with the radios